### DISCUSSION TITLE
Updates/DLC's.

### DISCUSSION THREAD

**Discussion by Jeff-2612** [skyline-emu/skyline#1361] [Ideas]
The Skyline team can implement in Skyline a method to install update/DLC's.

**Comment by mateoalfaro** [comment]
It's planned

**Comment by marcemira** [comment]
Any idea if we can make this happen at any time soon? Would be nice to not have to depend on hactools to generate to generate special all-in-one files, some times doesn't even works.

**Comment by Roguigo** [comment]
I hope they can do that, because I want to update Hollow knight and put the dlc god master but I don't know how to do that
